server/views/layout.pug

```diff
@@ -33,7 +33,7 @@ html(lang='en')
               collaborators: 'Collaborators',
               screenshot: 'Screenshot'
             },
-            lighthousekeys: [ 'seo', 'best-practices', 'accessibility', 'pwa', 'performance'],
+            lighthousekeys: [ 'seo', 'best-practices', 'accessibility', 'pwa', 'performance']
```

Comment:
I think there's just a missing comma here. Once I added one locally everything was working great!

Reply:
Thanks. I had a merge conflict debacle today. Thank you.